igvc_msgs/msg/motor_control.msg

```diff
@@ -0,0 +1,4 @@
+Header header
+uint8 left
+uint8 right
+duration duration
```

Comment:
Is the first `duration` supposed to be capitalized?

Reply:
Nope. Check [here](http://wiki.ros.org/msg#Fields)